Task_DEV_6/CustomStackTests/CustomStackTest.cs

```diff
@@ -0,0 +1,104 @@
+using NUnit.Framework;
+using System;
+using CustomStack;
+
+namespace CustomStackTest
+{
+    [TestFixture]
+    public class CustomStackTest
+    {
+        [Test]
+        public void IsEmptyTrueTest()
+        {
+            CustomStack<int> stack = new CustomStack<int>();
+            Assert.True(stack.IsEmpty());
+        }
+
+        [Test]
+        public void IsEmptyFalseTest()
+        {
+            CustomStack<int> stack = new CustomStack<int>();
+            stack.Push(7);
+            Assert.False(stack.IsEmpty());
+        }
+
+        [TestCase(5)]
+        [TestCase("abc")]
+        [TestCase(5.2)]
+        [TestCase('o')]
+        public void PopTest<T>(T item)
+        {
+            CustomStack<T> stack = new CustomStack<T>();
+            stack.Push(item);
+            int firstCount = stack.Count;
+            T element = stack.Pop();
+            int lastCount = stack.Count;
+            Assert.AreEqual(element, item);
```

Comment:
What is the different between `element` and `item`? Bad names.

Reply:
Done.